examples/cloud-router-2-virtual-device-connection/variables.tf

```diff
@@ -0,0 +1,68 @@
+variable "equinix_client_id" {
+  description = "Equinix client ID (consumer key), obtained after registering app in the developer platform"
+  type        = string
+}
+variable "equinix_client_secret" {
+  description = "Equinix client secret ID (consumer secret), obtained after registering app in the developer platform"
+  type        = string
+}
+variable "connection_name" {
+  description = "Connection name. An alpha-numeric 24 characters string which can include only hyphens and underscores"
+  type        = string
+}
+variable "connection_type" {
+  description = "Defines the connection type like VG_VC, EVPL_VC, EPL_VC, EC_VC, IP_VC, ACCESS_EPL_VC"
+  type        = string
+  default     = ""
```

Comment:
It seems that this can't be optional. So we shouldn't let it have a default value. Re-review your variables and make sure that we are only giving defaults to variables that aren't necessary to make the example work.